packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -48,6 +48,8 @@ contract DeployConfig is Script {
     uint256 public l2GenesisBlockGasLimit;
     uint256 public gasPriceOracleOverhead;
     uint256 public gasPriceOracleScalar;
```

Comment:
same, does it even make sense to read these in still?

Reply:
I was concerned with backwards compatibility but we could just say that the software no longer supports the older values per this commit which i think is actually how we want to do it to reduce tech debt